src/backend/distributed/citus--5.1-1--5.1-2.sql

```diff
@@ -0,0 +1,6 @@
+CREATE FUNCTION master_multi_shard_modify(text)
```

Comment:
Maybe master_modify_multiple_shards? 

btw, this function should be in the pg_catalog schema.